python/lsst/ts/ess/sensors/command_handler.py

```diff
@@ -193,6 +193,18 @@ async def stop_sending_telemetry(self):
             self._ess_instruments.remove(ess_instrument)
         return ResponseCode.OK
 
+    async def _process_sensor_telemetry(self, telemetry):
+        """Process incoming telemetry and pass it on to the callback coroutine.
```

Comment:
What does the callback coroutine do with it? Is this data then sent to the client or what?

I think the code would be clearer if you explained where the data was going and what it means to "process" the telemetry.

Reply:
I expanded the doc string.